.travis.yml

```diff
@@ -1,58 +1,24 @@
 language: go
+os: windows
 go:
   - "1.10.x"
-  - "1.11.x"
-  - "1.12.5"
   - "1.12.x"
+  - "1.13.x"
   - "1.x"
-  - "master"
```

Comment:
ACK on adding 1.13.x.  Any reason you removed 1.11.x, 1.12.5, and master?  (1.12.5 is there because it's the exact release that ncdns-repro inherits from tor-browser-build.)  Also if it's easy, maybe add 1.14.x since that's a thing now (though that's not a blocker for this PR)?

Reply:
in that case, may want to keep 1.12.5 around. Adding that back in. (re: ncdns-repro)

removed 1.11.x because it seemed redundant (having go 1.10x)

removed master because travis kept complaining about:
```
Your build has been stopped.
I don't have any idea what to do with 'master'.
  (using download type 'auto')
Failed to run gimme
```
([build log](https://travis-ci.org/github/aerth/certinject/jobs/681170182))

which is really strange because I usually have no issue with `master` branch travis runs, maybe because windows buildpack... also I tried `tip` and that didn't work either. (normally works fine on linux runs?)

regarding 1.14.x, we already have 1.x which currently is `go version go1.14.2` apparent in the travis log, so nothing would be different about the 1.x and 1.14.x runs

Reply:
> removed 1.11.x because it seemed redundant (having go 1.10x)

I've seen cases before where the build breaks for some set of old Go versions (e.g. a dependency breaks something); having 1.11.x present makes it easy to figure out which Go versions that breakage is confined to.

> removed master because travis kept complaining about:

Interesting.  Okay.

> regarding 1.14.x, we already have 1.x which currently is `go version go1.14.2` apparent in the travis log, so nothing would be different about the 1.x and 1.14.x runs

1.x will shift to 1.15.x once that's stable, at which point we'll lose test coverage of 1.14.x unless we remember to add it immediately.  Adding 1.14.x means that we'll have complete coverage until 1.16.x comes out, so we get a bit of extra leeway.

Reply:
Any objection to re-adding 1.11.x?

Reply:
no, i will add that back , too :)